pkg/config/tlscfg/ciphersuites.go

```diff
@@ -19,6 +19,14 @@ import (
 	"fmt"
 )
 
+// https://pkg.go.dev/crypto/tls#pkg-constants
+var versions = map[string]uint16{
+	"VersionTLS10": tls.VersionTLS10,
+	"VersionTLS11": tls.VersionTLS11,
+	"VersionTLS12": tls.VersionTLS12,
+	"VersionTLS13": tls.VersionTLS13,
```

Comment:
we didn't have much choice with cipher names, but here we have full control over names, and "VersionTLS13" seems like unnecessary complicated naming. We can simply use strings like "1.1", ..., "1.3".